README.md

```diff
@@ -11,7 +11,7 @@ EVCC is an extensible EV Charge Controller with PV integration implemented in [G
 ## Features <!-- omit in toc -->
 
 - simple and clean user interface
-- multiple [chargers](#charger): Wallbe, Phoenix (includes ESL Walli), go-eCharger, NRGkick (direct Bluetooth or via Connect device), SimpleEVSE, EVSEWifi, KEBA/BMW, openWB, Mobile Charger Connect, Fritz!DECT outlets, Tasmota outlets and any other charger using scripting
+- multiple [chargers](#charger): Wallbe, Phoenix (includes ESL Walli), go-eCharger, NRGkick (direct Bluetooth or via Connect device), SimpleEVSE, EVSEWifi, KEBA/BMW, openWB, Mobile Charger Connect, Smart-Home outlets with e-meter feature (Fritz!DECT, Tasmota, TP-Link) and any other charger using scripting
```

Comment:
willst du Smarthome outlets vllt in eine neue Zeile nehmen? e-meter feature würde ich streichen- mir ist nicht klar was damit gemeint ist

Reply:
> willst du Smarthome outlets vllt in eine neue Zeile nehmen?

Ok - mach ich

> e-meter feature würde ich streichen- mir ist nicht klar was damit gemeint ist

siehe unsere Diskussion https://github.com/andig/evcc/discussions/870 zum Thema fester Power Wert und Schwerpunkt auf Steckdosen, die auch messen können ... 
Aber ich nehm's mal raus, dann sind wir offener.